src/test-support/mocks/src/zone-ingresses/_/_overview.ts

```diff
@@ -57,27 +62,41 @@ export default ({ fake, env }: EndpointDependencies): MockResponder => (req) =>
                 id: fake.string.uuid(),
                 controlPlaneInstanceId: fake.hacker.noun(),
                 ...fake.kuma.connection(item, i, arr),
-                generation: 409,
-                status: {
-                  lastUpdateTime: '2021-07-13T09:03:11.614941842Z',
-                  total: {
-                    responsesSent: `${fake.number.int(30)}`,
-                    responsesAcknowledged: `${fake.number.int(30)}`,
-                  },
-                  cds: {
-                    responsesSent: `${fake.number.int(30)}`,
-                    responsesAcknowledged: `${fake.number.int(30)}`,
-                  },
-                  eds: {
-                    responsesSent: `${fake.number.int(30)}`,
-                    responsesAcknowledged: `${fake.number.int(30)}`,
-                  },
-                  lds: {
-                    responsesSent: `${fake.number.int(30)}`,
-                    responsesAcknowledged: `${fake.number.int(30)}`,
-                  },
-                  rds: {},
-                },
+                generation: fake.number.int({ min: 1, max: 500 }),
+                status: (() => {
+                  const xcks = fake.number.int({ min: 100, max: 500 })
+                  const stats = Object.entries(fake.kuma.partitionInto(
+                    {
+                      cds: Number,
+                      eds: Number,
+                      lds: Number,
+                      rds: Number,
+                    }, xcks,
+                  ))
+                  return {
+                    lastUpdateTime: '2021-07-13T09:03:11.614941842Z',
```

Comment:
Should we use a random fake date in the past? (same for the other cases)

Reply:
Yeah defo, I think (from vague recollection) the reason dates have been an outlier in the mocks is that faker doesn't offer a helper that formats dates like these golang formatted dates. Lemme take a quick look and see if thats correct. We should add one of our own formatters if so and use it everywhere.

Reply:
Oh! I just looked so there's code for this in that `fake.kuma.connection` thing we were chatting over the other day. I can extract that so we can use it here also, will be much better. I'll PR that separately in a sec.

Thanks for the suggestion! 🎉

Reply:
I made this https://github.com/kumahq/kuma-gui/pull/3158, gonna resolve this convo

Reply:
Perfect 👍 🙂